pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -907,8 +894,11 @@ func (oe *operationExecutor) generateUnmountDeviceFunc(
 		}
 		refs, err := attachableVolumePlugin.GetDeviceMountRefs(deviceMountPath)
 		if err != nil || len(refs) > 0 {
+			if err == nil {
+				err = errors.New("There are still mouting references for the device")
```

Comment:
`mouting` -> `mounting`

Might as well make this error more descriptive and include the device mount path and refs:

```GO
err = fmt.Errorf("...", deviceMountPath, refs)
```